Spellcheck pt. 2 (#6437)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/news_feed/creating_a_news_feed_item.md

You can re-order your cards within the News Feed page.
- Cards in the feed are ordered first by whether or not they have been seen by the user, unseen items are at the top of the feed.
  - A card is considered read if it has received an impression in the feed.

>   - Impressions are only counted if the card is viewable in the feed (i.e., if a user does not scroll down to read a card, an impression is not counted).

Author: ```suggestion
  - Impressions are only counted if the card is viewable in the feed (that is, if a user does not scroll down to read a card, an impression is not counted).
```
